recipes/rdflib/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "rdflib" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "eb02bd235606ef3b26e213da3e576557a6392ce103efd8c6c8ff1e08321608c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - rdfpipe = rdflib.tools.rdfpipe:main
+    - csv2rdf = rdflib.tools.csv2rdf:main
+    - rdf2dot = rdflib.tools.rdf2dot:main
+    - rdfs2dot = rdflib.tools.rdfs2dot:main
+    - rdfgraphisomorphism = rdflib.tools.graphisomorphism:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - isodate
+    - pyparsing
+
+  run:
+    - python
+    - isodate
+    - pyparsing
+    - html5lib
+    - sparqlwrapper
+
+test:
+  imports:
+    - rdflib
+    - rdflib.extras
+    - rdflib.plugins
+    - rdflib.plugins.parsers
+    - rdflib.plugins.parsers.pyMicrodata
+    - rdflib.plugins.parsers.pyRdfa
+    - rdflib.plugins.parsers.pyRdfa.extras
+    - rdflib.plugins.parsers.pyRdfa.host
+    - rdflib.plugins.parsers.pyRdfa.rdfs
+    - rdflib.plugins.parsers.pyRdfa.transform
+    - rdflib.plugins.serializers
+    - rdflib.plugins.sparql
+    - rdflib.plugins.sparql.results
+    - rdflib.plugins.stores
+    - rdflib.tools
+
+  commands:
+    - rdfpipe --help
+    - rdfgraphisomorphism --help
+
+  requires:
```

Comment:
Why is `networkx` required here, but is not a `run` requirement? I see that it is a test requirement in the `setup.py`, but since you aren't running the test quite and are only doing `import` style existence checks, I would think that you don't need it. Also, I don't think you need `html5lib` here if it is a `run` requirement.